Sources/Responses/EventType.swift

```diff
@@ -452,6 +452,36 @@ public enum EventType: BoxEnum {
             self = .itemsRemovedFromGroup
         case "FILE_WATERMARKED_DOWNLOAD":
             self = .watermarkedFileDownloaded
+        case "GROUP_CREATION":
+            self = .createdGroup
+        case "GROUP_DELETION":
+            self = .deletedGroup
+        case "GROUP_EDITED":
+            self = .editedGroup
+        case "EDIT_USER":
+            self = .editedUser
+        case "ADMIN_LOGIN":
+            self = .adminLogin
+        case "ADD_DEVICE_ASSOCIATION":
+            self = .addedDeviceAssocation
+        case "CHANGE_FOLDER_PERMISSION":
+            self = .changeFolderPermission
+        case "FAILED_LOGIN":
+            self = .failedLogin
+        case "LOGIN":
+            self = .login
+        case "REMOVE_DEVICE_ASSOCIATION":
+            self = .removedDeviceAssociation
+        case "DEVICE_TRUST_CHECK_FAILED":
+            self = .deviceTrustCheckFailed
+        case "TERMS_OF_SERVICE_ACCEPT":
+            self = .termsOfServiceAccepted
+        case "TERMS_OF_SERVICE_REJECT":
+            self = .termsOfServiceRejected
+        case "DOWNLOAD":
+            self = .downloaded
+        case "DELETE_USER":
+            self = .deletedUser
```

Comment:
Is this related to fixing existing tests?  If not, maybe we should break this out into another PR for adding missing EventTypes?

Reply:
I think that it should be placed it this commit. 
These values that we see here acutally were added a long time ago, but thanks to tests I found out that they are actually not working.
This is because there were misssing a mapping json(string) values into enum values, so I've just added it here.